calcom/cal.com
# fix: Routing Form - Show responses in the order of fields

## What does this PR do?

Fixes #11146

Before: https://www.loom.com/share/05bcdf1518844b21901cf57496cbf9b6
After: https://www.loom.com/share/137fc7b0664a4f9a898800b95b83b43a

## Type of change

## How should this be tested?
- Are there environment variables that should be set? No
- What are the minimal test data to have? Create a Routing Form with 3-4 fields.
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR - Make sure to enable 'Send Email to Owner' toggle for the form

## How should this be tested?
- Create a Routing Form with 3-4 fields. Submit it and observe the order of responses in the email that you get.

## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
